examples/nnet-and_gate.rs

```diff
@@ -41,7 +42,7 @@ fn main() {
 
     let layers = &[2, 1];
     let criterion = BCECriterion::new(Regularization::L2(0.));
-    let mut model = NeuralNet::new(layers, criterion, StochasticGD::default());
+    let mut model = NeuralNet::mlp(layers, criterion, StochasticGD::default(), Sigmoid);
```

Comment:
This isn't heavily related to your work but I think an inline comment here explaining what we're creating is a good idea.

i.e. `// Create a new multilayer perceptron with Stochastic gradient descent and Sigmoid activation`